Update organization.md - Underperformance policy (#1089)
flowforge/website: src/handbook/peopleops/organization.md

## Underperformance Policy

> We, at FlowFuse are committed to creating an environment where both our company and employees work together to maintain high performance standards. Our goal is to support our team members and managers in achieving these standards through clear and regular communication across all aspects of our organization.

Reviewer: ```suggestion
We're committed to creating an environment where both our company and employees work together to maintain high performance standards.
Our goal is to support our team members and managers in achieving these standards through clear and regular communication across all aspects of our organisation.
```
